<Window x:Class="FooEditor.ExceptionDialog"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        Title="ExceptionDialog" Height="300" Width="300">
    <StackPanel>
        <TextBlock TextWrapping="Wrap" Text="例外が発生しました。続行を押した場合は処理が続行され、終了を押した場合は保存後に終了します"/>
        <TextBox Height="200" Text="{Binding Message,Mode=OneWay}" IsReadOnly="True" ScrollViewer.VerticalScrollBarVisibility="Auto" ScrollViewer.HorizontalScrollBarVisibility="Auto">
        </TextBox>
        <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Margin="0,5,0,0">
            <Button Content="続行" Margin="0,0,5,0" Click="Button_Click_Continue"/>
            <Button Content="終了" Margin="0,0,5,0" Click="Button_Click_Exit"/>
        </StackPanel>
    </StackPanel>
</Window>
